db/migrate/20220309073750_change_users_table.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUsersTable < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :admin, :boolean
+    #Ex:- add_column("admin_users", "username", :string, :limit =>25, :after => "email")
+  end
```

Comment:
- [ ] According to the requirement for this project, it's required to add a `role` column to the users table but you're adding the `admin` column to the users table instead. Kindly add a `role` column to the users table instead of `admin`.

```suggestion
  def change
    add_column :users, :role, :string
  end
```

Reply:
Role is added